rdt/hyper_transformer.py

```diff
@@ -10,39 +10,69 @@
 
 
 class HyperTransformer(object):
-    """ This class is responsible for formatting the input table in a way
-    that is machine learning friendly
+    """Multitable transformer.
+
+    Arguments:
+        meta_file(str): Path to the meta.json file
+        table_dict(dict): Dict with the table structure (table_name -> (data, meta))
+
+    Returns:
+        HyperTransformer
+
+    The main propouse of the HyperTransformer class is to easily manage transformations
+    for a whole dataset.
+
+    It's required that one of `table_dict` or `meta_file` to be not null. In case both were
+    present `meta_file` takes precedence over `table_dict`.
     """
 
-    def __init__(self, meta_file=None):
-        """ initialize preprocessor """
+    def __init__(self, meta_file=None, table_dict=None):
```

Comment:
@csala Changes have been made to  use a single argument. The attribute hasn't been changed for two reasons:
- It will indeed break the uniformity with other parts of the projects.
- It's not the best name, as `table_dict` contains not only the metadata, but also the contents of each table, so calling it `metadata` could lead to confussion.